.github/workflows/login-onedrive-manual.yml

```diff
@@ -1,71 +1,50 @@
-          - name: Checkout repo content
-            uses: actions/checkout@v3       # checkout the repository content to github runner.
-            with:
-              token: ${{ secrets.WORKFLOW_TOKEN }}
-          - name: Setup Python
-            uses: actions/setup-python@v2
-            with:
-                python-version: 3.9.6
-          - name: Setup dependencies
-            run: |
-              pip install python-dotenv
-              pip install playwright
-              playwright install firefox
-              pip install pandas
-          - name: Clone private repo folder
-            env:
-              WORKFLOW_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
-            run: |
-              git clone \
-              --depth 1 \
-              --filter=blob:none \
-              --sparse \
-              https://$WORKFLOW_TOKEN@github.com/schmwong/login-log \
-              ;
-              cd login-log
-              git sparse-checkout init --cone
-              git sparse-checkout set onedrive
-              cd ..
-          - name: Run Python script
-            run: |
-              export USR_ONEDRIVE_1=${{ secrets.USR_ONEDRIVE_1 }}
-              export PWD_ONEDRIVE_1=${{ secrets.PWD_ONEDRIVE_1 }}
-              export USR_ONEDRIVE_2=${{ secrets.USR_ONEDRIVE_2 }}
-              export PWD_ONEDRIVE_2=${{ secrets.PWD_ONEDRIVE_2 }}
-              python keep-onedrive-active.py 
-          - name: Commit and push folder to private repo if updated
-            env:
-              WORKFLOW_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
+          - name: Get folder name from current workflow name
+            id: getname
             run: |
-              cd login-log
-              datestamp=$(date -u +%F)
-              git config user.name "${GITHUB_ACTOR}"
-              git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-              git add ./onedrive
-              if ! git diff-index --quiet HEAD; then
-                git commit -m "updated OneDrive logs for ${datestamp} (manual)" -- ./onedrive 
-                git push "https://${GITHUB_ACTOR}:$WORKFLOW_TOKEN@github.com/schmwong/login-log.git" \
-                HEAD:main || exit 0
-              fi
+              IFS=-
+              read -ra Arr <<< ${{ github.workflow }}
+              folder=${Arr[1]}
+              echo "folder name is $folder"
+              echo "::set-output name=foldername::$folder"
+              
+              
+  call-workflow:
+    needs: get-folder-name
+    uses: schmwong/keep-accounts-active/.github/workflows/reusable-autolog.yml@first-reusable-workflow
+    with:
+      folder: ${{ needs.get-folder-name.outputs.output1 }}
+      platform-name: OneDrive
+    secrets:
+      WORKFLOW_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
+      USR_ONEDRIVE_1: ${{ secrets.USR_ONEDRIVE_1 }}
+      PWD_ONEDRIVE_1: ${{ secrets.PWD_ONEDRIVE_1 }}
+      USR_ONEDRIVE_2: ${{ secrets.USR_ONEDRIVE_2 }}
+      PWD_ONEDRIVE_2: ${{ secrets.PWD_ONEDRIVE_2 }}
```

Comment:
`secrets: inherit` did not work.
Had to explicitly pass `secrets` to the reusable workflow.